okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -447,21 +441,23 @@ private void waitTenantInit(Tenant tenant, ModuleInstance getInstance,
    * Announce permissions for a set of modules to a permissions module.
```

Comment:
```suggestion
   * Conditionally announce permissions for a set of modules to a permissions module.  This will only happen if the permissions module is being enabled for the first time, not when it's upgraded.
```

Reply:
I'll add it.. Just realize that it may be called on last refresh and fooled by mdFrom = null (you probably saw that already)